recipes/repo.rb

```diff
@@ -277,6 +277,39 @@
   members ['pulp']
   action [:manage]
 end
+group 'gpg-vault' do
+  append true
+  members ['pulp']
+  action [:manage]
+end
+
+execute 'gpg-init' do
+  command 'gpg -K'
+  env 'HOME' => '/home/pulp'
+  user 'pulp'
+  group 'pulp'
```

Comment:
```suggestion
  user 'pulp'
  group 'pulp'
  environment 'HOME' => '/home/pulp'
```
Corrected `env` to `environment` although I would not be shocked to learn that `env` just works anyway.
The order is not significant but the following execute resources order the attributes user, group, environment so I maintain that for consistent reading.